advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+
+1. Open Connect CDC MonCon.
+
+2. As the Model is updated in the Connect CDC Director to add the `Replication Distribution`, click on the `Model Update`, this will add the `Replication Distribution` to the Connect CDC MonCon Interface.
+
+3. The replication request will be displayed once the `Model` is updated successfully.
+
+![Replication Request](Images/replicationrequest3.png)
+
+4. In the `Request` Section, select the `Distribution Model` created for Replication and right click and select `Start`.
+
+![Distribution Model](Images/distrmodel.png)
+
+5. On the `Source Instance`, update the data to be replicated in the `Source Tables`.
+
+6. On the `Target Instance`, check data in the `Target Tables`, data will be updated there.
+
+![pgAdmin](Images/pgAdmintargettables.png)
+
+## Data Transformation
+
+Precisely Connect CDC has options that let you modify the source data you are distributing or even create or derive the data you distribute. You can use predefined functions to accomplish these data transformations or you can construct your own operations or functions.
+
+For example, to express source currency values in terms of a different currency, you specify the transformation algorithm to use for that column mapping. For some common transformations, the algorithm is supplied as a predefined Connect CDC function.
+
+The algorithms you specify are known in the Connect CDC Director as expressions. The Connect CDC Expression Handler parses them and then creates runtime code that calculates the result for the expressions.
+
+In addition to or instead of calls to predefined functions, expressions may have an arbitrary number of constants, column references, and calculations on those constants and columns and functions. Their results must be compatible in type and length with the target column.
+
+### Steps to demonstrate data transformations
+
+The following steps demonstrate data transformations.
+
+1. Open Connect CDC Director.
+
+2. Go to `Source DBMS Server` —> `Tables` —> `Sending Tables`.
+
+3. Select the table(s) to perform data transformation using the checking box in front of them and right click and select `Properties` —> `Mappings`.
+
+4. On the `Mapping` tab, you define the column mappings for each of your target tables with respect to the source tables. The `Mapping` tab displays the corresponding columns in the `Target Column` and `Source Data` columns in each row. Connect CDC Director automatically mapped target columns to source columns that have the same name.
+
+When the Connect CDC Director maps a source table to a same-named target table, it also associates individual source table columns with same-named target table columns. You then select which of these column pairings you want to include in your data distribution, which additional column pairings you want to arrange, and what type of data transformation, if any, you want to assign to each column pairing.
+
+Mappings Tab has the following options
```

Comment:
For some reason the items under this are in code form. Like a block. Is that intentional?

Reply:
yes its intentional.